packages/quantic/force-app/main/default/lwc/quanticResultsPerPage/quanticResultsPerPage.js

```diff
@@ -48,12 +59,30 @@ export default class QuanticResultsPerPage extends LightningElement {
     this.hasResults = this.searchStatus.state.hasResults;
   }
 
+  validateChoicesDisplayed() {
+    return this.choicesDisplayed.split(',').map((choice) => {
+      const parsedChoice = parseInt(choice, 10);
+      if (isNaN(parsedChoice)) {
+        throw new Error(`The choice value "${choice}" from the "choicesDisplayed" option is not a number.`);
+      }
+
+      return parsedChoice;
+    });
+  }
+
+  validateInitialChoice() {
+    if (!this.initialChoice) {
+      return;
+    }
+    if (!this.choices.includes(Number(this.initialChoice))) {
+      throw new Error(`The "initialChoice" option value "${this.initialChoice}" is not included in the "choicesDisplayed" option "${this.choicesDisplayed}".`);
```

Comment:
Do we really want to throw? We could just Warn but then set the initialChoice to 10?

Reply:
We could. I feel it might be a little too subtle and thus confusing to default to another behaviour. If i were a dev user I would likely make the jackie chan "wtf?" face before I noticed the warning.